modules/ROOT/pages/6.0-release-notes-plugins.adoc

```diff
@@ -0,0 +1,300 @@
+
+[[changed-plugins-print]]
+==== Print
+
+`print` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `print` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+[[spellchecker]]
+==== Spellchecker
+
+As per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the `spellchecker` plugin has been removed. It is no longer part of {productname} 6.0.
+
+The https://tiny.cloud/tinymce/features/#productivity[Premium plugin], https://tiny.cloud/docs/plugins/premium/tinymcespellchecker/[Spellchecker Pro], which offers equivalent functionality and more, is available.
+
+[[tabfocus]]
+==== TabFocus
+
+The `tabfocus` plugin has been removed. It is no longer part of {productname} 6.0. And it is not otherwise available.
+
+The `tabfocus_elements` option, which relied on the `tabfocus` plugin, no longer functions.
+
+As well, the `tab_focus` option was previously deprecated by `tabfocus_elements`. With this release, the `tab_focus` option has been removed completely.
+
+As per standard web practice, the `tabindex` attribute should be used instead of the `tabfocus` plugin or any of the `tabfocus` plugin’s options.
+
+As per that standard web practice, the `tabindex` attribute is copied from the target element to the iframe.
+
+[[changed-plugins-table-of-contents]]
+==== Table of contents
+
+The `toc` plugin is no longer part of the Core open source editor. _Table of Contents_ is now a https://tiny.cloud/tinymce/features/#productivity[Premium plugin].
+
+NOTE: although there are no end-user facing changes, as part of the general architectural change, the `tableofcontents` Premium plugin has modernised its output. It uses the ES2020 library and targets ES2018. It also uses https://terser.org[terser] to improve performance.
+
+[[text-color]]
+==== Text color
+
+Text color’s functionality has been a built in part of {productname} since version 5.
+
+Consequently, and as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[announcement], the empty `textcolor` plugin has now been removed.
+
+[[text-pattern]]
+==== TextPattern
+
+The `textpattern` functionality is no longer a plugin. It is now a Core part of {productname} 6.0.
+
+In {productname} 6.0, there is no setting or specifying this functionality in the `plugins` option.
+
+All references to `textpattern` should be removed from your plugin configuration. Leaving references to removed plugins in a plugin configuration will cause console errors documenting the plugin failing to load.
+
+Also, the API that called `textpattern` has been removed.
```

Comment:
This is another place where we could mention that the API has been replaced with calls to `editor.options.get('text_patterns')` and `editor.options.set('text_patterns', newPatterns)`.

Reply:
The text pattern API wasn't actually ever documented, so we possibly shouldn't actually even mention it here.

Reply:
That is a good point

Reply:
```suggestion
```